packages/pipeline-editor/src/PropertiesPanel/CustomFormControls/StringArrayControl.tsx

```diff
@@ -1,18 +1,123 @@
-export class StringArrayControl {
+import React from "react";
+
+export class StringArrayControl extends React.Component {
```

Comment:
`StringArrayControl` shouldn't extend anything. This isn't an actual react component, `renderControl` just returns a react component